docs/reference/commandline/start.md

```diff
@@ -22,6 +22,85 @@ Start one or more stopped containers
 
 ## Examples
 
+### Start a container
+
+To start a container, specify the container ID or image name. For example:
```

Comment:
Oh! My bad, missed this one; when starting a container, you specify the name (or ID) of the _container_, not the image.

```suggestion
To start a container, specify the container name or ID. For example:
```